src/errors.rs

```diff
@@ -54,8 +48,7 @@ impl<E: StdError + Deserialize + Serialize + Send + 'static> fmt::Display for Er
 impl<E: StdError + Deserialize + Serialize + Send + 'static> StdError for Error<E> {
     fn description(&self) -> &str {
         match *self {
-            Error::ClientDeserialize(_) => "The client failed to deserialize the server response.",
-            Error::ClientSerialize(_) => "The client failed to serialize the request.",
+            Error::ClientSerialize(_) => "The client failed to (de)serialize the request.",
```

Comment:
Request or response. So maybe "The client failed to serialize the request or deserialize the response."?